code/modules/religion/sects/shadow_sect.dm

```diff
@@ -319,73 +321,52 @@
 	if(!rite_target)
 		return FALSE
 	rite_target.AddComponent(/datum/component/anti_magic, MAGIC_TRAIT, _magic = TRUE, _holy = FALSE)
-	//glowing wings overlay
-	to_chat(rite_target, "<span class='userdanger'>You are grateful to have been converted to the dark by [user]. Serve [user.real_name], and assist [user.p_them()] in completing [user.p_their()] goals at any cost.</span>")
+	var/obj/effect/proc_holder/spell/targeted/shadowwalk/SW = new
+	rite_target.AddSpell(SW)
 	playsound(rite_target, 'sound/weapons/fwoosh.ogg', 75, 0)
 	rite_target.visible_message("<span class='notice'>[rite_target] has been blessed by the rite of [name]!</span>")
+	if(rite_target == user || !user.mind.special_role)
+		return
+	to_chat(rite_target, "<span class='userdanger'>You are grateful to have been converted to the dark by [user]. Serve [user.real_name], and assist [user.p_them()] in completing [user.p_their()] goals at any cost.</span>")
+	var/objective = "You are grateful to have been converted to the dark by [user]. Serve [user.real_name], and assist [user.p_them()] in completing [user.p_their()] goals at any cost!"
+	brainwash(rite_target, objective, "Shadow Conversion")
```

Comment:
I don't get the point that people won't be brainwashed when they just don't have `special_role`. the variable `special_role` does nothing in fact, but also if you intended it to be antag-only, still I don't think giving it to antag only works.

* Remove the early return

and I disagree with giving such objective directly stating `serve chaplain` to people.
* Reword the objective message more less compelling:
  * My suggestion: `"Bring the world the true darkness, let the world bear the true comfort, deeply, in the faithful way of the shadow."`

The reason I don't like "Serve chaplain" is that it 100% antagonizes people, and only dedicated to the chaplain. This can lead them to go into nothing shadow related, and I don't think making people 100% chaplain side is good.
So, I suggested making the message a bit toned down and vague, to allow people  "can be chaplain side" instead of "must be chaplain side"

Reply:
Might be mis-reading that first line, but afaik all antag roles add a `special_role`, so it check if the user(chaplain) has any form of special role, and if they're not, we dont brainwash, so if a regular crewmate gets access to the sect they cant brainwash people

I agree with the objective message tho, adjusting it